library-v3/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If we're copying over everything in `/res`, then these files in `library-v3/src/main/res/` should be deleted.